flow/include/flow/EncryptUtils.h

```diff
@@ -72,12 +74,29 @@ typedef enum {
 	ENCRYPT_HEADER_AUTH_TOKEN_MODE_NONE = 0,
 	ENCRYPT_HEADER_AUTH_TOKEN_MODE_SINGLE = 1,
 	ENCRYPT_HEADER_AUTH_TOKEN_MODE_MULTI = 2,
-	ENCRYPT_HEADER_AUTH_TOKEN_LAST = 3 // Always the last element
+	ENCRYPT_HEADER_AUTH_TOKEN_LAST = std::numeric_limits<uint8_t>::max() // Always the last element
```

Comment:
Idea was LAST to touch the limits, also helps in future extensions, otherwise, LAST values would need to be adjusted. Both scheme would work, but I don't see any issues with current proposal, do we?

Reply:
like, `isEncryptHeaderAuthTokenAlgoValid` etc will allow values that's not recognizable by current version, which doesn't seem like desired behavior.